### ISSUE TITLE
FreeBSD support

### ISSUE THREAD

**Issue by mfyuce** [ftao/python-ifcfg#29]
This library can easily support FreeBSD, I think. Waht do you say?

**Comment by benjaoming** [comment]
Thanks for asking

Yes, definitely -- someone (with FreeBSD!) is definitely encouraged to pick up this PR... https://github.com/ftao/python-ifcfg/pull/23

(feel welcome to supersede the PR with your own branch)

**Comment by rgeorgia** [comment]
Just forked this repo. I'll do this. Not to "up" on nose, but I'll learn. I'll add support for the major BSDs.

**Comment by rgeorgia** [comment]
Excuse my ignorance. Not sure if this is the platform for questions and clarification. I have ifcfg working with FreeBSD and NetBSD (there's a little cliche with OpenBSD). I know almost nothing about nose. If I create a venv with python3.10 nostetests fail with a traceback. 

`  File "/usr/home/rgeorgia/workspace/python_projects/python-ifcfg/venv/lib/python3.10/site-packages/nose/suite.py", line 106, in _set_tests
    if isinstance(tests, collections.Callable) and not is_suite:
AttributeError: module 'collections' has no attribute 'Callable'
`

All tests pass with python3.8. The tox.ini file has python3.8 as the max version. Are we staying with python3.8?

**Comment by benjaoming** [comment]
This sounds more like a Python 3.10 issue that should be fixed.

**Comment by benjaoming** [comment]
It seems like `collections.abc.Callable` would work in all Python versions, maybe you can check to see if it does on Python 3.10, too?

**Comment by rgeorgia** [comment]
Will do! Once I finish the tests (which include learning about nose) I just issue a pull request, correct? I forked the project and created a branch called freebsd.

**Comment by benjaoming** [comment]
Yes, I'm here to do a new release whenever needed, and @ftao is maybe also on the line :)
